dlp/inspect_content_test.py

```diff
@@ -381,55 +387,61 @@ def test_inspect_gcs_multiple_files(bucket, topic_id, subscription_id, capsys):
         topic_id,
         subscription_id,
         ["EMAIL_ADDRESS", "PHONE_NUMBER"],
-    )
+        timeout=1)
 
     out, _ = capsys.readouterr()
 
     assert "Inspection operation started" in out
     # Cancel the operation
-    operation_id = out.split("Inspection operation started: ")[1].split("\n")[0]
+    operation_id = out.split(
+        "Inspection operation started: ")[1].split("\n")[0]
     print(operation_id)
     client = google.cloud.dlp_v2.DlpServiceClient()
     client.cancel_dlp_job(operation_id)
 
 
-@flaky
-def test_inspect_datastore(datastore_project, topic_id, subscription_id, capsys):
-    @eventually_consistent.call
-    def _():
-        inspect_content.inspect_datastore(
-            GCLOUD_PROJECT,
-            datastore_project,
-            DATASTORE_KIND,
-            topic_id,
-            subscription_id,
-            ["FIRST_NAME", "EMAIL_ADDRESS", "PHONE_NUMBER"],
-        )
+def test_inspect_datastore(
+        datastore_project, topic_id, subscription_id, capsys):
+    inspect_content.inspect_datastore(
+        GCLOUD_PROJECT,
+        datastore_project,
+        DATASTORE_KIND,
+        topic_id,
+        subscription_id,
+        ["FIRST_NAME", "EMAIL_ADDRESS", "PHONE_NUMBER"],
+        timeout=1)
 
-        out, _ = capsys.readouterr()
-        assert "Info type: EMAIL_ADDRESS" in out
+    out, _ = capsys.readouterr()
+    assert "Inspection operation started" in out
+    # Cancel the operation
+    operation_id = out.split(
+        "Inspection operation started: ")[1].split("\n")[0]
+    print(operation_id)
+    client = google.cloud.dlp_v2.DlpServiceClient()
+    client.cancel_dlp_job(operation_id)
 
 
-@flaky
 def test_inspect_datastore_no_results(
-    datastore_project, topic_id, subscription_id, capsys
-):
-    @eventually_consistent.call
-    def _():
-        inspect_content.inspect_datastore(
-            GCLOUD_PROJECT,
-            datastore_project,
-            DATASTORE_KIND,
-            topic_id,
-            subscription_id,
-            ["PHONE_NUMBER"],
-        )
-
-        out, _ = capsys.readouterr()
-        assert "No findings" in out
-
-
-@pytest.mark.skip(reason="unknown issue")
+        datastore_project, topic_id, subscription_id, capsys):
+    inspect_content.inspect_datastore(
+        GCLOUD_PROJECT,
+        datastore_project,
+        DATASTORE_KIND,
+        topic_id,
+        subscription_id,
+        ["PHONE_NUMBER"],
+        timeout=1)
+
+    out, _ = capsys.readouterr()
+    assert "Inspection operation started" in out
+    # Cancel the operation
```

Comment:
Should we move this functionality into a function? we're doing it in a couple of different places

Reply:
Sure, done